sputnikdao-factory2/src/lib.rs

```diff
@@ -24,31 +40,41 @@ pub struct SputnikDAOFactory {
 impl SputnikDAOFactory {
     #[init]
     pub fn new() -> Self {
+        assert!(!env::state_exists(), "The contract is already initialized");
```

Comment:
This line is curious to me…
Note that in this update: https://github.com/near/near-sdk-rs/blob/master/CHANGELOG.md#301-03-25-2021
We have:

>BREAKING #[init] now checks that the state is not initialized. This is expected behavior. To ignore state check you can call #[init(ignore_state)]

and I believe it happens here:
https://github.com/near/near-sdk-rs/blame/5dee86c9db5ca6ae30630d1f73fbe960f0e364b5/near-sdk-macros/src/core_impl/code_generator/impl_item_method_info.rs#L89-L91

But also, there are three places in the `near-sdk-macros` directory that have a similar check, but yours has the `!` in front of it, negating it. I'm not seeing that in the macros.

The two others:
https://github.com/near/near-sdk-rs/blob/5dee86c9db5ca6ae30630d1f73fbe960f0e364b5/near-sdk-macros/src/core_impl/code_generator/item_impl_info.rs#L497-L499 and 

and

https://github.com/near/near-sdk-rs/blob/5dee86c9db5ca6ae30630d1f73fbe960f0e364b5/near-sdk-macros/src/core_impl/code_generator/item_impl_info.rs#L579-L581

Reply:
no, you're actually right. The check is no longer necessary since we have `init` macro. I've removed it, thanks for the suggestion!